gapic/src/main/com/google/gapid/views/TracerDialog.java

```diff
@@ -440,17 +440,6 @@ public AndroidInput(
         traceTarget.addBoxListener(SWT.Modify, targetListener);
         targetListener.handleEvent(null);
 
-        Listener apiListener = e -> {
```

Comment:
Are we confident enough marking this as "works" for GLES, or do we want to mark it as experimental?

Reply:
Because as soon as the button works, people will start sending bugs.

Reply:
Done.

Reply:
Done.